### PR TITLE
Contao Release Stages Bundle metdata - BROCKHAUS AG

### PR DESCRIPTION
**Pull request by INoTime**

### CODE REVIEW

File: linter/allowlists/default.txt [hunk 1]
```diff
@@ -350,3 +350,13 @@ WebP
...
+release
+Release
+Releases
+releasing
+stages
+Stages
+auto
+deployment
+Prod
+Stage
```
Comment (fritzmg) [inline linter/allowlists/default.txt:362]: I do not think all of these are necessary. For example `releasing` does not exist in the German title or description, so I don't think it needs to be specifically allowed for all languages?

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: linter/allowlists/en.txt [hunk 2]
```diff
@@ -203,3 +203,9 @@ Watchlist
 watchlist
 yes
 youtube
+auto
+deployment
+release
+Releases
+stages
+Stage
```
Comment (fritzmg) [inline linter/allowlists/en.txt:211]: Some of these are regular English words, so they do not need to be on the English allow list.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: linter/allowlists/en.txt (hunk 2 above)
Comment (INoTime) [reply linter/allowlists/en.txt:211]: So i have to commit these words to de.txt? Because some of the words are used in de like "Release" or "Stage"

Comment (fritzmg) [reply linter/allowlists/en.txt]: Yes, `Releases` and `Stage` need to be added to the `de` allow list:

```
Error: ] [Package: brockhaus-ag/contao-release-stages-bundle; Language: de]:    
         Property "description" does not pass the spell checker. Either update  
         the allow list or fix the spelling :) Errors: Releases, Stage
```

**Review by fritzmg — APPROVED** [review]

**Pull request merged**